project-management/README.md

```diff
@@ -0,0 +1,95 @@
+
+We use Github [Issues](https://help.github.com/articles/about-issues/), [Milestones](https://help.github.com/articles/about-milestones/) and [Projects](https://help.github.com/articles/about-project-boards/) to track and mange the work for the organization
+
+* [Project](#Projects)
+* [Milestones](#Milestones)
+* [Issues](#Issues)
+* [Pull Requests](#Pull%20Requests)
+
+## Projects
+
+Projects are used to track complicated work in a single repository, or to coordinate work across multiple repositories
+
+Only two types of projects are allowed: **Releases** and **Features**
+
+### Why?
+
+Provides an easy to track long lived features or cross repository task
+
+### Releases
+
+* Correlate to VSCode marketplace releases
+* Encompass all work done for that release
+* Can contain issues/prs from just one, or more than one repo
+
+### Features
+
+* Do not correlate to any release, can span releases
+* Contains issues and PRs that encompass work done in more than one repository
+* Does not contain issues/prs that are only in one repo
+* Can live longer than a Release project
+
+
+## Milestones
+
+Milestones are used to track the work of a single release of a repository
+
+### Why?
+
+Provides an easy reference when creating a release
+
+## Issues
+
+Each Issue will be tagged for a milestone. Each issue _may_ be tagged with one or more projects, for example, documentation errors would not be assoicated with a Project but will still have a milestone.
+
+### Why?
+
+Provides an easy reference when creating a release
+
+## Pull Requests
+
+Work/Code/Changes will be introduced into repositories using Pull Requests (PRs). Work should not be directly merged into the default, usually master, branch.
+
+* Each PR will be tagged with one or more Issues
+* Each PR will be tagged for a milestone
+* Each PR _may_ be tagged with one or more projects, for example, documentation errors would not be assoicated with a Project but will still have a milestone.
+
+### Why?
+
+Provides a high level overview of work in progress and completed
+
+---
+
+## Examples
+
+If we have a collection of Projects, Repositories, Milestones, Issues, and Pull Requests
+
+<img align="center" width="462" height="346" alt="Lingua Pupuli Logo" src="images/base.png">
+
+---
+
+### A maintence change
+
+A maintenance or documentation change which isn't related to a project would look like the following;
+
+<img align="center" width="462" height="346" alt="Lingua Pupuli Logo" src="images/maint-change.png">
+
+---
+
+### A complicated feature project
+
+A feature that is complicated and will contain one or more issues would look like the following;
+
+<img align="center" width="462" height="346" alt="Lingua Pupuli Logo" src="images/long-live.png">
+
+---
+
+### A cross repository project
+
+A feature that spans across multiple projects would look like the following;
```

Comment:
It would be useful, I think, to clarify that the releases of these two projects may not be tied together  (ie, releasing A doesn't _necessarily_ depend on releasing B), but full implementation of the feature **does**.